### PR TITLE
Fixed the throughput burst issue

### PR DESCRIPTION
**Pull request by yanjunz97**
This PR fixes the throughput burst issues. 

Previously we reset `totalCount` and `deltaCount` static elements after the record is sent. As the throughput is calculated by `diff(octetTotalCount) / diff(flowEndSeconds)`, the existing `octetTotalCount` should not be reset to 0. 

The PR also adds reset of throughput after the the record is sent. This deals with the condition that a record is sent by both active and inactive expiry interval. The receiver will get two records with the same `flowEndSecond`. In this case, the record sent by inactive expiry interval timeout should have the throughput as 0.

### CODE REVIEW

**Comment by yanjunz97** [comment]
I added a commit to refactor the code which includes two changes:

- Reseting throughput is put in the same function which resets the statElements, as I did not come up with the case that these two functions may be called separately.
- Using a loop instead of repeating code to reset the value.

I think these changes may not have influences on performance but makes the code cleaner. But if there is any concern regarding the  change, I can revert it.

File: pkg/intermediate/aggregate.go [hunk 1]
```diff
@@ -640,28 +643,37 @@ func (a *AggregationProcess) aggregateRecords(incomingRecord, existingRecord ent
...
+func (a *AggregationProcess) ResetStatAndThroughputElementsInRecord(record entities.Record) error {
 	statsElementList := a.aggregateElements.StatsElements
 	antreaSourceStatsElements := a.aggregateElements.AggregatedSourceStatsElements
 	antreaDestinationStatsElements := a.aggregateElements.AggregatedDestinationStatsElements
 	for i, element := range statsElementList {
-		if ieWithValue, _, exist := record.GetInfoElementWithValue(element); exist {
-			ieWithValue.ResetValue()
-		} else {
-			return fmt.Errorf("element with name %v in statsElements is not present in the record", element)
+		// TotalCount static elements should be skipped as they are used in the thoughput calculation.
```
Comment (dreamtalen) [inline pkg/intermediate/aggregate.go:654]: static -> statistic

**Review by dreamtalen — COMMENTED** [review]
LGTM, thanks yanjun fixing this issue.

**Review by heanlan — APPROVED** [review]
Haven't review yet. Test approve.

File: pkg/intermediate/aggregate.go (hunk 1 above)
Comment (heanlan) [inline pkg/intermediate/aggregate.go:654]:
```suggestion
		// TotalCount statistic elements should not be reset to zeroes as they are used in the
		// throughput calculation. 
```

Comment (yanjunz97) [reply pkg/intermediate/aggregate.go]: Done

Comment (yanjunz97) [reply pkg/intermediate/aggregate.go]: Done

**Review by heanlan — APPROVED** [review]
LGTM

**Review by heanlan — APPROVED** [review]

**Pull request merged**